source/user-manual/api/rbac/configuration.rst

```diff
@@ -6,22 +6,25 @@ Configuration
 =============
 Before modifying RBAC policies, roles and more, it is necessary to log into the API. You can find a detailed guide on how to do it within the :ref:`Getting started <api_getting_started>` section of the API.
 
-The endpoints required to manage RBAC belong to the *Security* group. To run them, this configuration guide uses the API interface. However, the command-line ``cURL`` can be used instead as explained in :ref:`Getting started <api_getting_started>`.
-
 Set RBAC mode
 -------------
-You can modify the RBAC mode (which we mentioned in the :ref:`How it works <api_rbac_how_it_works>` section). This is a security configuration so it is necessary to use the security endpoint ``/security/config``, specifically the ``PUT`` method. Just write the following setting inside the request body of the endpoint, setting the desired mode (white/black).
+You can modify the RBAC mode (which we mentioned in the :ref:`How it works <api_rbac_how_it_works>` section). This is a security configuration so it is necessary to use the security endpoint ``/security/config``, specifically the ``PUT`` method. To do so, you can run the following cURL command, replacing your token and the desired mode (white/black).
+
+.. code-block:: console
+
+    # curl -X PUT "https://localhost:55000/v4/security/config?pretty=true" -H "Authorization: Bearer <YOUR_JWT_TOKEN>" -d "{\"rbac_mode\":\"<DESIRED_RBAC_MODE>\"}"
 
 .. code-block:: json
+    :class: output
 
     {
-      "mode": "white"
+       "message": "Configuration successfully updated"
     }
 
 You can also change other settings (not related to RBAC) such as the duration of the JWT tokens, as well as check the current configuration with the ``GET`` method or restore the default one with the ``DELETE`` method.
 
 .. note::
-    Wazuh API needs to be restarted for the changes to take effect.
+    All tokens are revoked for security reasons when changing RBAC mode. It will be necessary to obtain a new token after the change.
```

Comment:
```suggestion
    All tokens are revoked for security reasons when the RBAC mode is changed. It will be necessary to log in and obtain a new token after the change.
```